### DISCUSSION TITLE
Default C value for SVM

### DISCUSSION THREAD

**Discussion by bagustris** [felixbur/nkululeko#147] [General]
Is there any particular reason why setting C_val to `0.001` for svm model?

https://github.com/felixbur/nkululeko/blob/0d26c2e5ea2f2bf973137428b9bb6557e6c4518d/nkululeko/models/model_svm.py#L16

Because in experience, the default scikit-learn C value, i.e., 1.0, always (if not most) obtains better performance than the default one.

**Comment by felixbur** [comment]
thanks for noting, you're right and i must have misread this somehow, thinking that's the "official" default.
Please fix this in you upcoming pull request
